docs/en/query_language/functions/hash_functions.md

```diff
@@ -310,11 +310,11 @@ SELECT murmurHash3_128('example_string') AS MurmurHash3, toTypeName(MurmurHash3)
 Calculates `xxHash` from a string. It is proposed in two flavors, 32 and 64 bits.
 
 ```sql
-SELECT xxHash32('srting');
+SELECT xxHash32();
```

Comment:
What a hell is that? :) What a hash does it return when called w/o params? And why it differs from empty string? :) 

```
SELECT     xxHash32(),     xxHash32(''),     xxHash64(),     xxHash64('');

Row 1:
──────
xxHash32():   4263699484
xxHash32(''): 46947589
xxHash64():   16324913028386710556
xxHash64(''): 17241709254077376921
```

Reply:
Done

Reply:
Replaced `xxHash32()`  → `xxHash32('')`.